sklearn/decomposition/_nmf.py

```diff
@@ -625,8 +632,10 @@ def _multiplicative_update_w(X, W, H, beta_loss, l1_reg_W, l2_reg_W, gamma,
     return delta_W, H_sum, HHt, XHt
 
 
-def _multiplicative_update_h(X, W, H, beta_loss, l1_reg_H, l2_reg_H, gamma):
-    """Update H in Multiplicative Update NMF."""
+def _multiplicative_update_h(X, W, H, A, B, beta_loss, l1_reg_H, l2_reg_H,
+                             slice_index, gamma, rho):
+
+    """update H in Multiplicative Update NMF"""
```

Comment:
I know this is a private function, but as the number of parameter grows, we might want to start detailing the parameters in the docstring.

Reply:
Done.